cfpb/cfgov-refresh
# Quick fix Spanish translation errors for the Financial Well-Being Scale's /about/ page

Found after deployment

## Changes

- Text updated for a more accurate translation, per stakeholder request

## Testing

1. Load this branch on your machine, and visit http://localhost:8000/consumer-tools/financial-well-being/about/
2. Check that the first line of the body content starts with "En el CFPB trabajamos para ayudar" instead of "En el CFPB trabajamos ayudar"
3. Check that the second sentence of the second paragraph starts with "El bienestar financiero quiere decir, cómo su situación" instead of "El bienestar financiero quiere decir como su situación"

## Checklist
